src/components/ButtonGroup/ButtonGroup.module.css

```diff
@@ -3,31 +3,37 @@
 \*------------------------------------*/
 
 /**
- * 1) A group of buttons displayed in a horizontal fashion
+ * A group of buttons displayed in an organized fashion.
  */
 .button-group {
-  display: flex;
-  align-items: center;
-  gap: var(--eds-size-2);
+  display: inline-grid;
+  grid-auto-flow: column;
 }
 
 /**
- * Responsive button group
- * 1) A button group that's stacked by default then moves side-by-side
- *    when space becomes available 
+ * Spacing variants.
  */
-.button-group--responsive {
-  @media all and (max-width: $eds-bp-md) {
-    align-items: flex-start;
-    flex-direction: column;
-  }
```

Comment:
Mmm.. I think I'd want to see it in context. Like what kind of UI were these buttons being used in and how many were there? I personally feel like I would rather have them wrap.

Reply:
I feel the same about wrapping. I converted to `display: inline-flex` and added a `wrap` prop to control wrapping behavior. Let me know how you feel about it since it was originally implemented as grid.

Reply:
Ack pushed a wrong branch (jlee/ButtonGroup not jlee/Buttongroup), pushed this branch now

Reply:
When would we want the `'nowrap'` or `'wrap-reverse'` cases (or was that for illustration purposes)? I think it'd be enough to always `wrap` and if people need, they can override the style

Reply:
I'm down to always `wrap`. Was trying to allow more options, but definitely do agree most cases would just use `wrap`. One case I can think of for `nowrap` is when the intention is for the users to scroll the buttons to prevent them from taking up too much space on the screen, but as you said, devs can override if necessary

Reply:
hmm yeah I see -- not sure we want to encourage that design tho. and should be rare in any case

Reply:
I like the updated solution of allowing it to wrap whenever it gets too small, regardless of the screen it's on. It feels very flexible and maintainable 👍